database/oracle/README.md

```
@@ -0,0 +1,30 @@
+
+### Build and Run the Docker Image:
+1. **Build the Docker image:**
+```bash
+docker build -t oracle-db .
+```
+
+2. **Run the Docker container:**
```

Markdownlint

Expected: 1; Actual: 2; Style: 1/1/1
Ordered list item prefix

(MD029, ol-prefix)